.babelrc

```diff
@@ -1,4 +1,3 @@
 {
-  "presets": ["env", "react"],
-  "plugins": ["transform-object-rest-spread", "transform-class-properties"]
-}
\ No newline at end of file
+  "presets": [["cozy-app", { "node": true }], "react"]
```

Comment:
I had an issue using the app version with the `babel-plugin-transform-runtime` breaking the CSP.
Let's use the node preset for now until I figured out on my side.

Reply:
Any news about this point ?

Reply:
Nope :/, I completely forgot this point